README.md

```diff
@@ -137,6 +138,7 @@ This will run:
 2. `rails importer:import_answers` - imports the initial practice data the Diffusion Marketplace team collected via Survey Monkey, images and all~
 3. `rails importer:initial_featured` - sets the first three initial featured practices for the homepage
 4. `rails diffusion_history:all` - set up the initial diffusion history for the first five practices. Individual commands can be found here:  `lib/tasks/diffusion_history.rake`
+5. `rails milestones:milestone_transfer` - transfers original timeline entry milestones to the new milestone model
```

Comment:
this is suppose to be 'milestones:milestones_transfer' (the extra 's' in the second milestone)

Reply:
Updated